mne/io/nirx/nirx.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, fname, preload=False, verbose=None):
 
         # Check that the file format version is supported
         if not any(item == hdr['GeneralInfo']['NIRStar'] for item in
-                   ["\"15.0\"", "\"15.2\""]):
+                   ["\"15.0\"", "\"15.2\"", "\"15.3\""]):
```

Comment:
We could make this more readable by writing ` ['"15.0"', '"15.2"', '"15.3"']` (enclose strings with single quotes, then you don't have to escape double quotes).

Reply:
And wouldn't it be sufficient to write:

`if hdr['GeneralInfo']['NIRStar'] not in ['"15.0"', '"15.2"', '"15.3"']:`